### PR TITLE
Fix up the Siteboss571 discovery yaml to split pdnOutputCurrentValue and pdnMainCurrentValue indexes

### PR DESCRIPTION
**Pull request by jediblair**
Updated the SiteBoss571 discovery module to account for index overlap in the current tables - the per-module ( pdnMainCurrentValue ) values were taking precedence over the per-port ( pdnOutputCurrentValue ) values for id 1.1 and 200.1.
Added index overrides for these two OIDs.

I've also tidied up the descriptions a little. 

This is a slightly breaking change in that the current RRD file names will change, so historic current data will disappear from the web interface, I couldn't see a nice way around this - if keeping the index name for the pdnOutputCurrentValue then 1.1 and 200.1 files would be corrupted by the pdnMainCurrentValue data in the existing files, so just opted to go with index overrides on both.

DO NOT DELETE THE UNDERLYING TEXT

#### Please note

> Please read this information carefully. You can run `./lnms dev:check` to check your code before submitting.

- [x] Have you followed our [code guidelines?](https://docs.librenms.org/Developing/Code-Guidelines/)
- [ ] If my Pull Request does some changes/fixes/enhancements in the WebUI, I have inserted a screenshot of it.
- [x] If my Pull Request makes discovery/polling/yaml changes, I have added/updated [test data](https://docs.librenms.org/Developing/os/Test-Units/).

#### Testers

If you would like to test this pull request then please run: `./scripts/github-apply <pr_id>`, i.e `./scripts/github-apply 5926`
After you are done testing, you can remove the changes with `./scripts/github-remove`.  If there are schema changes, you can ask on discord how to revert.

### CODE REVIEW

File: tests/snmpsim/siteboss571_siteboss571.snmprec [hunk 1]
```diff
@@ -1,6 +1,6 @@
-1.3.6.1.2.1.1.1.0|4|\"SiteBoss 571 2.11.380 STD
```
Comment (PipoCanaja) [inline tests/snmpsim/siteboss571_siteboss571.snmprec:1]: Hi @jediblair 
Could you please create a new snmprec file instead of changing the existing ?

**Review by PipoCanaja — CHANGES_REQUESTED** [review]

**Comment by PipoCanaja** [comment]
Hi @jediblair 
Thanx for your PR and welcome in the project. Your PR seems all right, we just need to tidy up the tests part and you'll be good to go. 

snmprec file is a kind of "snapshot" of the device. And the json is the result of the processing of the snmprec. So any change in the code that breaks discovery or polling can be tracked (hopefully) if json content changes unexpectedly. 

We can create multiple variants of snmprec (different versions, etc etc) and that's what would be the best in your case : create a new variant, compute the matching JSON for this variant, and re-compute the existing (old) snmprec to validate the change in the JSON. 

If you have issue with the JSON part (as it seems to be currently), I'll help you with that.

**Comment by jediblair** [comment]
Thanks, I've added the new snmprec file and new test output. 
Additionally today I noticed this has a stub OS detection php file as well as the yaml , all this seems to accomplish is the detection constantly flipping the system name from the sysName to the siteName from the MIB with a couple of accompanying log entries every time. I don't think this is needed in the system name field as in all our other devices this is just the hostname, rather than a long location field. (Which is captured in location anyway)
FWIW the other Siteboss OS detections have a corresponding PHP file, but i didn't want to mix up the 550 and 530 with the 571 here. Might it be worth me removing those in a seperate PR?

**Comment by PipoCanaja** [comment]
Hi @jediblair 
Makes sense concerning the siteName removal. Let stick with 571 here, and you can open a new PR for the others. I am now fixing the tests and if all looks good, this PR will be ready to merge.

**Pull request merged**